src/runners/helpers/db.py

```diff
@@ -166,17 +192,31 @@ def sql_value_placeholders(n):
     return ", ".join(["(%s)"] * n)
 
 
+def insert(table, values, ovewrite=False):
```

Comment:
We should fix the typoed param name here so that we don't need to reproduce the typo every time we want to overwrite data in a table.

Reply:
looks fixed